steamclock/version-check-kotlin
# Have DefaultDialog automatically launch Play Store for the user (#14)

### Related Issue: #14 


### Summary of Problem:
Update button current does nothing on default dialog handler; would be nice to have it automatically open the Play Store page for the given application when clicked.

### Proposed Solution:
`DefaultUpgradeDialog` takes packageName and will now attempt to launch the Play Store for that package

### Testing Steps:
Launch the sample app and click on the Update button; the Play Store app should launch. Because there is no listing for the sample app this will come up empty, but I tested this out by manually setting the packageName to be one of our public apps and the store page did load.
